tokio/src/runtime/queue.rs

```diff
@@ -34,17 +34,17 @@ pub(super) struct Inject<T: 'static> {
 pub(super) struct Inner<T: 'static> {
     /// Concurrently updated by many threads.
     ///
-    /// Contains two `u8` values. The LSB byte is the "real" head of the queue.
-    /// The `u8` in the MSB is set by a stealer in process of stealing values.
+    /// Contains two `u16` values. The LSB byte is the "real" head of the queue.
```

Comment:
I think it might be worth documenting someplace that the indices are 16 bits because they need to be larger than the queue's capacity, to help guard against re-introducing the bug.